File: LogicalProgramming/PrimeNumber.cs
using System;
using System.Collections.Generic;

namespace ConsoleApp1
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Enter the number:");
            int n =Convert.ToInt32( Console.ReadLine());
            int count = 0;
            for (int i=1;i<=n;i++)
            {
                if (n%i ==0)
                {
                    count++;
                }
            }
            if (count==2)
            {
                Console.WriteLine("{0} is prime Number",n);
            }
            else
            {
                Console.WriteLine("{0} is Not Prime Number",n);
            }
            Console.ReadLine();

        }
    }

}

File: JavaScript/PrimeNumber.js
function printPrime()
{
    var n=7;
    var count = 0;
    for (var i=1;i<=n;i++)
    {
        if (n%i ==0)
        {
            count++;
        }
    }
    if (count==2)
    {
        console.log("{0} is prime Number",n);
    }
    else
    {
        console.log("{0} is Not Prime Number",n);
    }
         
}
printPrime();
